.github/workflows/e2e.yml

```diff
@@ -23,13 +23,15 @@ jobs:
     runs-on: ${{ matrix.platform }}
     steps:
       - name: Checkout code
+        if: matrix.go != 'tip' || matrix.platform != 'windows-latest'
```

Comment:
I guess we can skip this by now as long as we are testing with stable version.

Reply:
I've created an issue to track this: https://github.com/grafana/xk6-browser/issues/1002